sovrn-htb.js

```diff
@@ -26,6 +26,7 @@ var SpaceCamp = require('space-camp.js');
 var System = require('system.js');
 var Network = require('network.js');
 var Utilities = require('utilities.js');
+var { version } = require('./package.json');
```

Comment:
This destructuring assignment can break on older browsers.  Removing this line will maintain compatibility with older browsers.  Importing package.json is not necessary in order to access the adapter version number.